components/automate-ui/src/app/entities/automate-settings/automate-settings.model.ts

```diff
@@ -1,73 +1,141 @@
 export class JobSchedulerStatus {
-  running: boolean;
   jobs: IngestJob[];
 
-  constructor(running: boolean, ingestJobs: IngestJob[]) {
-    this.running = running;
+  constructor(ingestJobs: IngestJob[]) {
     this.jobs = ingestJobs;
   }
-
-  getJob(name: string): IngestJob {
-    return this.jobs.find((job: IngestJob) => job.name === name);
-  }
 }
 
 export interface ConfigureSettingsRequest {
   jobs: IngestJob[];
 }
 
 export interface RespJob {
-  running: boolean;
   name: string;
-  every: string;
+  nested_name?: string;
+  disabled: boolean;
+  recurrence: string;
   threshold: string;
-  last_run: Date;
-  next_run: Date;
-  last_elapsed: Date;
-  started_on: Date;
+  purge_policies: any; // needs specific type
+  last_elapsed?: Date;
+  next_due_at?: Date;
+  last_enqueued_at?: Date;
+  last_started_at?: Date;
+  last_ended_at?: Date;
 }
 
 export interface RespJobSchedulerStatus {
-  running: boolean;
-  jobs: RespJob[];
+  infra: {
+    jobs: RespJob[];
+  };
+  compliance: {
+    jobs: RespJob[];
+  };
+  event_feed: {
+    jobs: RespJob[];
+  };
+  services: {
+    jobs: RespJob[];
+  };
 }
 
 // IngestJobs is an enum that defines the list of jobs that the
 // JobScheduler has inside the ingest-service
 export enum IngestJobs {
-  // MissingNodes: Checks when a node hasn't check-in
-  // for a period of time
-  MissingNodes = 'missing_nodes',
+  //  EventFeed
+  EventFeedRemoveData = 'eventFeedRemoveData',
+  EventFeedServerActions = 'eventFeedServerActions',
 
-  // MissingNodesForDeletion: Checks when a node has been missing
-  // for a period of time
-  MissingNodesForDeletion = 'missing_nodes_for_deletion',
+  // Service Groups
+  ServiceGroupNoHealthChecks = 'serviceGroupNoHealthChecks',
+  ServiceGroupRemoveServices = 'serviceGroupRemoveServices',
+
+  // Client Runs
+  ClientRunsRemoveData = 'clientRunsRemoveData',
+  ClientRunsLabelMissing = 'clientRunsLabelMissing',
+  ClientRunsRemoveNodes = 'clientRunsRemoveNodes',
 
-  // DeleteNodes: Removes completely from elasticsearch nodes that
-  // have been marked for deletion
-  DeleteNodes = 'delete_nodes'
+  // Compliance
+  ComplianceRemoveReports = 'complianceRemoveReports',
+  ComplianceRemoveScans = 'complianceRemoveScans'
 }
 
 export class IngestJob {
-  running: boolean;
+  category: string;
   name: string;
+  nested_name?: string;
+  recurrence?: string;
   threshold: string;
-  every?: string;
-  lastRun?: Date;
-  nextRun?: Date;
-  lastElapsed?: Date;
-  startedOn?: Date;
+  disabled: boolean;
+  purge_policies?: any; // needs specific type
```

Comment:
Suggest add a TODO so it is more clear to everyone, plus it makes it easier to find such loose ends.
```suggestion
  purge_policies?: any; // TODO needs specific type
```